src/features/farming/animals/chickenMachine.ts

```diff
@@ -0,0 +1,178 @@
+import { CHICKEN_FEEDING_TIME } from "features/game/lib/constants";
+import { assign, createMachine, State } from "xstate";
+
+const TICK_INTERVAL = 1; // 1 second
```

Comment:
Should we increase this for production to 10 or 20 seconds? Do we forsee any lag/optimizations issues with this tick running for lots of chickings?

Reply:
We need to run on every second since the time to egg is based on time/seconds. I don't see this being an issue as the UI is not responding/aware of the tick at all.